frontend/packages/client/src/pages/Community.js

```diff
@@ -195,11 +196,25 @@ export default function Community() {
 
   const { instagramUrl, twitterUrl, websiteUrl, discordUrl } = community ?? {};
 
+  const titleClassNames = classnames(
+    "is-size-5 has-text-weight-bold",
+    { "mb-3": notMobile },
+    { "mb-1": !notMobile }
+  );
+  const memberClassNames = classnames(
+    "has-text-grey",
+    { "small-text mb-3": notMobile },
+    { "is-size-6 mb-1": !notMobile }
+  );
+  const headerContainerClassNames = classnames(
+    "is-flex container community-header section",
+    { "is-justify-content-space-between": notMobile }
+  );
   return (
     <section className="full-height pt-0">
       {community ? (
         <div className="is-flex community-header-wrapper">
-          <div className="is-flex container community-header section is-justify-content-space-between">
```

Comment:
is-justify-content-space-between can't be used on mobile since it stretches content: 
<img width="446" alt="Screen Shot 2022-06-14 at 12 29 54" src="https://user-images.githubusercontent.com/7526740/173617072-0a483378-5235-4d44-9dc0-18af124453b6.png">


after fix: 
<img width="424" alt="Screen Shot 2022-06-14 at 12 30 39" src="https://user-images.githubusercontent.com/7526740/173617099-53845528-688c-4788-9e7f-c8d7ef917f0e.png">